.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: true
+dist: trusty
+os:
+- linux
+language: node_js
+node_js:
+- '7'
+install:
+- node --version
+- npm --version
+- npm i
```

Comment:
Let's use `yarn` over `npm` as we're using that in the projects in general. 

@hunterlester you'll need to add yarn cache to the setup of the image also.